R/dropdown.R

```diff
@@ -149,7 +149,8 @@ selectInput <- function(inputId, label, choices, selected = NULL, multiple = FAL
 #' @param input_id The id of the input object
 #' @param choices All available options one can select from. If no need to update then leave as \code{NULL}
 #' @param choices_value What reactive value should be used for corresponding choice.
-#' @param value The initially selected value.
+#' @param value A value to update dropdown to. \code{character(0)} and \code{""} clear the dropdown.
+#' \code{NULL} (the default) does not change selection.
```

Comment:
Unfortunately, this is not true. For a multiple selection, with the value set to `NULL` and changed `choices` it will clear the selection (which makes sense, as the new choices might not include the old values). But for a single select dropdown, the same action will set the first choice as selected. IMO it should also clear the selection.

Reply:
Fixed in https://github.com/Appsilon/shiny.semantic/commit/dd0e434fe596a39221b7f5c3eef6410fa8104113